shared/js/dataDriver.js

```diff
@@ -284,6 +301,7 @@ loop.DataDriver = function() {
       }));
       this._sse.addEventListener("open", this);
       this._sse.addEventListener("put", this);
+      this._sse.addEventListener("error", this);
```

Comment:
Repeating comment from before: On a larger question.. do we actually want _connect to do all of what it has already done? E.g., refetching all of old chat?